configure.ac

```diff
@@ -49,6 +49,14 @@ AC_ARG_VAR([OMR_CROSS_CONFIGURE],
 	Compiler and feature detection tests will be invalid. All options must be configured manually.
 	One of: yes,no. (Default: no)])
 
+AC_ARG_VAR([OMR_PRODUCT_DESCRIPTION], [The description of product integerating OMR. (Default: OMR 1.0)])
```

Comment:
I agree, but this should be set in `AC_INIT`.

The help string should use the same macros we use to actually set the default values, namely `AC_PACKAGE_{NAME,VERSION,STRING,URL}`.